io-engine/src/grpc/v1/replica.rs

```diff
@@ -560,20 +563,24 @@ impl ReplicaRpc for ReplicaService {
                         match lvol.prepare_snap_config(
                             &args.snapshot_name,
                             &args.entity_id,
-                            &args.txn_id
+                            &args.txn_id,
+                            &args.snapshot_uuid
                         ) {
                             Some(snap_config) => snap_config,
                             None => return Err(LvsError::SnapshotConfigFailed {
                                 name: args.replica_uuid,
                                 msg: "tx id / snapshot name not provided".to_string(),
                             })
                         };
+                    let replica_uuid = lvol.uuid();
+                    let replica_size = lvol.size();
+                    info!("Prepare Snapshot for Replica {lvol:?}");
```

Comment:
Should this be debug?

Reply:
Missed to remove it. It was added for my testing. Will remove now.